index.html

```diff
@@ -0,0 +1,634 @@
+                    <button type="submit" class="material-icons">search</button>
+                </form>
+            </div>
+
+            <div id="buttons">
+                <div class="flex-container">
+                    <button class="material-icons">video_call</button>
+                </div>
+                <div class="flex-container">
+                    <button class="material-icons">apps</button>
+                </div>
+                <div class="flex-container">
+                    <button class="material-icons">sms</button>
+                </div>
+                <div class="flex-container">
+                    <button class="material-icons">notifications</button>
+                </div>
+                <div class="flex-container">
+                    <img id="image-profile" src="image/photo.jpg" alt="photo of user">
+                </div>
+            </div>
+        </div>
+    </header>
+
+
+    <main>
+
+        <div>
+            <div id="main_video">
+              <iframe width="560" height="315" src="https://www.youtube.com/embed/WV2Ed1QTst8"
+              frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen>
+              </iframe>
+            </div>
+
+            <div id="principal-info">
+                <h1>Contributing to Open Source Part I: The Easy Way</h1>
+                <div class="flex-container">
+                    <div>
+                        <span>487,995 views
+                        </span>
+                    </div>
+                    <div class="menu-info flex-container">
+                        <div class="flex-container">
+                            <button class="material-icons btn-info">thumb_up</button>
+                            <a href="#">340</a>
+                        </div>
+                        <div class="flex-container">
+                            <button class="material-icons btn-info">thumb_down</button>
+                            <a href="#">13</a>
+                        </div>
+                        <div class="flex-container">
+                            <button class="material-icons btn-info">reply</button>
+                            <a href="#">SHARE</a>
+                        </div>
+                        <div class="flex-container">
+                            <button class="material-icons btn-info">playlist_add</button>
+                            <a href="#">SAVE</a>
+                        </div>
+                        <div class="flex-container">
+                            <button class="material-icons btn-info">more_horiz</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+
+            <div id="description">
+                <div class="top-row flex-row flex-container">
+                    <div class="img-channel">
+                        <img src="image/channel-img.jpg" alt="Imagem do Canal">
```

Comment:
`SUGGESTION`

Consider adding the images you referenced. You seem to be adding images that are not available on the repo.